addon/hint/show-hint.css

```diff
@@ -1,6 +1,6 @@
 .CodeMirror-hints {
   position: absolute;
-  z-index: 10;
+  z-index: 1000;
```

Comment:
Is *1000* really necessary here? Can you give a little more detail on the need/reason for this change and the degree to which it is being changed?

Reply:
It needs to be higher than the windows which are at `100`. `1000` is a clean number above that. I can test if `999` or something else works, but this seems more reliably above everything.